### ISSUE TITLE
Continuous mapping in OpenStreetMap of paths top of existing ways.

### ISSUE THREAD

**Issue by sekerob** [osmItalia/cai_scripts#18]
We've been fighting this issue for a few months in Italy where mappers come in, and start drawing a path up and over every street, road, track, trail, stream, buildings to the extend there's sometimes even 3 paths on top of each other, which get blue flagged in OSM Inspector for instance. Since this has been going on for the last 4-6 months me thinketh it's a result of poor instructions introduced in 2022, the latest result a new mapper who added this one

https://www.openstreetmap.org/changeset/129523914

I know this is not the right place, but SOSEC seems to have their own regional divisions, where this is a nation wide problem from Veneto to Calabria. Please intervene, this has to stop.

**Comment by lucadelu** [comment]
Hi @sekerob, I will report to some people involved in SOSEC, could you please send me privately the username involved?

@dieterdreist could you report this to SOSEC too?

**Comment by dieterdreist** [comment]
Yes, have already reported it, sekerob informed me simultaneously.

**Comment by sekerob** [comment]
@lucadelu, if only i could find the PM function in GitHub (found discussions about this feature as a want though).

Here are the links I've preserved of a number of cases from conversations with several mappers in the OSM PM system:
https://www.openstreetmap.org/changeset/129523914
https://www.openstreetmap.org/changeset/129250698
https://www.openstreetmap.org/changeset/129100580
https://www.openstreetmap.org/changeset/129048881
https://www.openstreetmap.org/changeset/123223489
https://www.openstreetmap.org/changeset/125400183

I've noticed that first in list above was deleted without communication by a mapper (got name) and 3 routes on Sicily just were summarily deleted (got name), no public communication that I can discern by them. Can't see that as a way forward either. Also noticed a duplicate mapper then complaining in newly raised CS and drawing/importing over a path anew.
